### PR TITLE
Adding a basic configuration for the mention bot

### PR DESCRIPTION
**Pull request by DeepDiver1975**
https://raw.githubusercontent.com/facebook/mention-bot/master/CONFIG.md

### CODE REVIEW

**Comment by rullzer** [comment]
O nice that it is configurable!

:+1:

File: .mention-bot [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "maxReviewers": 3,
+  "numFilesToCheck": 5,
+  "alwaysNotifyForPaths": [
+    {
+      "name": "DeepDiver1975", 
+      "files": ["apps/dav/**"] 
+    }
+  ],
+  "userBlacklist": ["owncloud-bot"]
```
Comment (MorrisJobke) [inline .mention-bot]: Oh noooo :cry:

**Comment by MorrisJobke** [comment]
:+1:

**Comment by DeepDiver1975** [comment]
ok - let's see how this works out .....

**Pull request merged**